main_test.go

```diff
@@ -12,7 +12,11 @@ import (
 	"os/exec"
 	"path/filepath"
 	"reflect"
+	"sort"
+	"strings"
 	"testing"
+
+	"github.com/stretchr/testify/assert"
```

Comment:
please refrain from using `testify/assert` and just use `testing`, thanks.

Reply:
Question: Why do you think we should refrain from using testify/assert? I think it makes the job of writing unit tests much easier. Every time a block of code like below changes to a single line.
```
if err != nil {
   t.Fatalf(....)
}
```

I am assuming you have a valid reason and I am changing now.